test/jdk/java/util/TimeZone/TimeZoneBoundaryTest.java

```diff
@@ -339,7 +340,7 @@ static void testUsingMillis(Date d, boolean startsInDST)
 
       if (inDaylight != lastDST)
       {
-      logln("Switch " + (inDaylight ? "into" : "out of")
+      System.out.println("Switch " + (inDaylight ? "into" : "out of")
```

Comment:
I'd remove this commented-out code entirely.

Reply:
Removed this and the other occurrence you spotted.